src/samples/ChildBenefitsClaim/index.tsx

```diff
@@ -119,7 +118,14 @@ export default function ChildBenefitsClaim() {
   const { t } = useTranslation();
   let operatorId = '';
   
-
+  // On render reset the timeout functionality.
+  useEffect(()=>{
+    initTimeout(setShowTimeoutModal);
+    return () => {
+      clearTimeout(applicationTimeout);
+      clearTimeout(signoutTimeout);
+    }
+  },[applicationTimeout, signoutTimeout, setShowTimeoutModal,initTimeout])
```

Comment:
why initTimeout is dependency when we are not using inside useEffect?

Reply:
Just an oversight. I've removed all the dependencies as I only want this to run once.